doc/contributing/coding-conventions.chapter.md

```diff
@@ -643,7 +643,7 @@ Here are examples of package tests:
 
```

Comment:
This chapter should be in `nixos/doc`, as it is entirely NixOS-specific.

Reply:
I would agree that this aspect is at the interface between NixOS and Nixpkgs, but the _use case_ is entirely Nixpkgs-specific.
Package tests are a Nixpkgs concept. The implementation of such tests is quite often done by referring to NixOS tests, but that does not make them NixOS-specific. The focus is 1. the basics of linking those tests up, 2. dealing with version matrixes, which would otherwise make NixOS tests unsuitable.
I'm not entirely opposed to moving this, but it seems arbitrary. Did I miss something?

Reply:
Yeah, you're right. What really confuses me though is the cyclic dependency of `nixpkgs` on NixOS code here, but that is an unrelated issue.